.evergreen/ocsp/ocsp_mock.py

```diff
@@ -0,0 +1,46 @@
+#! /usr/bin/env python3
```

Comment:
Taken from https://github.com/mongodb/mongo/blob/master/jstests/ocsp/lib/ocsp_mock.py.

Reply:
Added the link as a comment to both python files!

Reply:
Did you push the changes? I don't see a link in the current diff. Feel free to resolve this once changes are reflected in the PR.

Reply:
Ah, sorry, was on a hotspot when I was pushing, something went wrong. Now it's truly pushed!